src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1084,6 +1100,30 @@ public void sendBattleRecordsToGameData(final IDelegateBridge aBridge) {
     }
   }
 
+
+  void fightStrategicBombingRuns(final IDelegateBridge delegateBridge) {
+    boolean bombing = true;
+    fightStrategicBombingRuns(delegateBridge, () -> getPendingBattleSites(bombing),
+            (territory, battleType) -> getPendingBattle(territory, bombing, battleType));
+  }
+
+  @VisibleForTesting
+  void fightStrategicBombingRuns(final IDelegateBridge delegateBridge, Supplier<Collection<Territory>> pendingBattleSiteSupplier,
+             BiFunction<Territory, BattleType, IBattle> pendingBattleFunction) {
+    // Fight all air and bombing fightStrategicBombingRuns without needing user to click on them.
+    // The rules say that bombing battles should be done first, so automatically do those combats.
+    for( final Territory t : pendingBattleSiteSupplier.get()) {
+      final IBattle airRaid = pendingBattleFunction.apply(t, BattleType.AIR_RAID);
+      if( airRaid != null ) {
+        airRaid.fight(delegateBridge);
+      }
+      final IBattle bombingRaid = pendingBattleFunction.apply(t, BattleType.BOMBING_RAID);
+      if( bombingRaid != null ) {
+        bombingRaid.fight(delegateBridge);
+      }
```

Comment:
What is the difference between an AIR_RAID and a BOMBING_RAID? Does the order of them matter?